core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -6,7 +6,7 @@
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
  *
- *    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
ditto

Reply:
I think it's because I use the wrong formatter in `Intellij`, what is the recommanded way of formatting Kafka code in `Intellij` ?